civitai-downloader-v2/scripts/import_summary.py

```
@@ -0,0 +1,157 @@
+                stats = raw_data.get('stats', {})
+                download_count = stats.get('downloadCount', 0) or 0
+                
+                if download_count > 0:
+                    popular_models.append({
+                        'id': row[0],
+                        'name': row[1][:50],
+                        'type': row[2],
+                        'category': row[3],
+                        'downloads': download_count
+                    })
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)